.github/workflows/windows-zip.yml

```diff
@@ -65,11 +65,12 @@ jobs:
     - name: Extract Etherpad
       run: 7z x etherpad-lite-win.zip -oetherpad
 
-    - name: list
-      run: dir etherpad
+    - name: Install Cypress
+      run: npm install cypress -g
```

Comment:
Why install it globally? Isn't superuser access required for that?

Reply:
so I can access the binary easily.  And we have enough privilege in CI